src/adapters/cf-worker/adapter.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  canonicalizeHeaders,
+  flatHeaders,
+  addHeader,
+  AdapterArgs,
+  NormalizedRequest,
+  NormalizedResponse,
+} from '../../runtime/http';
+
+interface WorkerAdapterArgs extends AdapterArgs {
+  rawRequest: Request;
+}
+
+export async function workerConvertRequest(
+  adapterArgs: WorkerAdapterArgs,
+): Promise<NormalizedRequest> {
+  const req = adapterArgs.rawRequest;
+  const body = await req.text();
+  const headers = {};
+  for (const [key, value] of req.headers.entries()) {
+    addHeader(headers, key, value);
+  }
```

Comment:
Why not use `canonicalizeHeaders` in the return statement (like the Node version), vs. this `for` loop with `addHeader`?